target/linux/ramips/dts/mt7621_iodata_wn-ax1167gr2.dts

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_iodata_wn-xx-xr.dtsi"
+
+/ {
+	compatible = "iodata,wn-ax1167gr2", "mediatek,mt7621-soc";
+	model = "I-O DATA WN-AX1167GR2";
+};
+
+&partitions {
+	partition@6b00000 {
+		label = "Backup";
+		reg = <0x6b00000 0x1480000>;
+		read-only;
+	};
+};
+
+&pcie1 {
```

Comment:
This is in DTS because you plan to add other devices without? Because on a quick look I see no difference between this and the other device.

Reply:
WN-AX2033GR/GR2 use pcie0 and pcie1, and the configuration will be different from WN-AX1167GR2 and WN-DX1167R.